### PR TITLE
Website modifications and add a mailmap

### PR DESCRIPTION
**Pull request by cticenhour**
Made a few adjustments to the website:

- Cleaned up a broken link for David Graves' old group at UC-Berkeley.
- Updated the INL HPC acknowledgement to reflect the current language.
- Updated the Zapdos BibTeX citation to a slightly better version in the bib files. 
- Added a `documentation.bib` file for BibTeX entries used in the documentation, and demonstrated usage of the MooseDocs BibTeX extension in the body of ShootMethodLog documentation. This bib file should be used for all similar in-text citations that are needed in the text in the future. The other bib files present in `doc/content/bib` are for the "Works that cite Zapdos" listings.

Also added a mailmap file to unify contribution statistics across multiple emails that have creeped into the repository. Without the file, contributors under the same name, but with a different email, would be counted as separate contributors.

@csdechant the MooseDocs demo is the one I mentioned in #227.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Alex Lindsay <adlinds3@ncsu.edu>
+Alex Lindsay <lindsayad@users.noreply.github.com>
+Alex Lindsay <alexlindsay239@gmail.com>
+Casey Icenhour <cticenho@ncsu.edu>
+Casey Icenhour <casey.icenhour@inl.gov>
+Casey Icenhour <cticehour@gmail.com>
+Shane Keniley <keniley1@illinois.edu>
+Corey DeChant <csdechan@ncsu.edu>
+Corey DeChant <37221357+csdechant@users.noreply.github.com>
+Corey DeChant <csdechan@ncsu.edu>
```
Comment (cticenhour) [inline .mailmap:10]:
```suggestion
Corey DeChant <csdechan@ncsu.edu>
Corey DeChant <37221357+csdechant@users.noreply.github.com>
```

Oopsie had a duplicate.

File: .mailmap [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Alex Lindsay <adlinds3@ncsu.edu>
+Alex Lindsay <lindsayad@users.noreply.github.com>
+Alex Lindsay <alexlindsay239@gmail.com>
+Casey Icenhour <cticenho@ncsu.edu>
+Casey Icenhour <casey.icenhour@inl.gov>
+Casey Icenhour <cticehour@gmail.com>
+Shane Keniley <keniley1@illinois.edu>
+Corey DeChant <csdechan@ncsu.edu>
+Corey DeChant <37221357+csdechant@users.noreply.github.com>
```
Comment (csdechant) [inline .mailmap:9]:
```suggestion
Corey DeChant <csdechan@ncsu.edu>
Corey DeChant <corey.dechant1@gmail.com>
Corey DeChant <37221357+csdechant@users.noreply.github.com>
```

Including my Gmail account.

File: .mailmap [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Alex Lindsay <adlinds3@ncsu.edu>
+Alex Lindsay <lindsayad@users.noreply.github.com>
+Alex Lindsay <alexlindsay239@gmail.com>
+Casey Icenhour <cticenho@ncsu.edu>
```
Comment (csdechant) [inline .mailmap:4]: @cticenhour Do you still have access to your NCSU email? I heard NCSU deactivated soon after someone graduates.

File: .mailmap [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+Corey DeChant <csdechan@ncsu.edu>
+Corey DeChant <37221357+csdechant@users.noreply.github.com>
+John Ryan Haase <jhaase1@nd.edu>
+John Ryan Haase <jhaase1@users.noreply.github.com>
+Cody Permann <codypermann@gmail.com>
+Cody Permann <cody.permann@inl.gov>
+Cody Permann <Cody.Permann@inl.gov>
+Derek Gaston <friedmud@gmail.com>
+Derek Gaston <derek.gaston@inl.gov>
+Grayson Gall <ggall713@gmail.com>
```
Comment (csdechant) [inline .mailmap:17]: @gsgall Do you want to include your NCSU email as a developer contact?

**Comment by csdechant** [comment]
@cticenhour Deleted some comments I made since I misread the propose of the mailmap file. Everything looks good to me.

**Review by csdechant — APPROVED** [review]

**Comment by cticenhour** [comment]
@csdechant @gsgall The mailmap is purely for statistics. If you only ever use one email with your GitHub commits, then there is no need to be on the mailmap. This is only to ensure that when I run `git shortlog --summary --numbered` we don't have three Alex Lindsays or two Corey DeChants! 😄

**Pull request merged**